internal/datanode/data_node.go

```diff
@@ -875,7 +886,10 @@ func (node *DataNode) Import(ctx context.Context, req *datapb.ImportTaskRequest)
 		log.Warn(msg)
 		importResult.State = commonpb.ImportState_ImportFailed
 		importResult.Infos = append(importResult.Infos, &commonpb.KeyValuePair{Key: "failed_reason", Value: msg})
-		reportFunc(importResult)
+		reportErr := reportFunc(importResult)
+		if reportErr != nil {
+			log.Warn("fail to report import state to root coord", zap.Error(err))
```

Comment:
Here and below:

1) Should be zap.Error(reportErr) ?

2) It is cleaner to merge the lines like:
```
		if err := reportFunc(importResult); err != nil {
			log.Warn("fail to report import state to root coord", zap.Error(err))
```